CMakeLists.txt

```diff
@@ -22,7 +22,9 @@ endif()
 
 if(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++latest")
```

Comment:
yeah, decided to not one line it in case anyone needs to remove it for whatever reason and so it's clearer to see separate flags